wails-frontend/src/pages/PasswordPromptHandler/PlainText/PlainText.tsx

```diff
@@ -19,33 +20,39 @@ export function PlainText(props: PromptRequestData) {
 
   return (
     <div className="flex flex-col items-center gap-4 mas-menu-default w-[326px]">
-      <div className="flex w-full items-center justify-between">
+      <div className="flex w-full justify-between">
         <p>{Intl.t('password-prompt.sign.from')}</p>
-        <p className="mas-caption">{maskAddress(WalletAddress)}</p>
+        <div className="flex flex-col">
+          <p className="mas-menu-default">{Nickname}</p>
+          <p className="mas-caption">{maskAddress(WalletAddress)}</p>
+        </div>
       </div>
-      <hr className="h-0.25 bg-neutral opacity-40 w-full" />
 
       {Description && (
-        <div className="flex flex-col w-full h-fit">
-          <AccordionCategory
-            isChild={false}
-            iconOpen={<FiChevronDown />}
-            iconClose={<FiChevronUp />}
-            customClass={'!p-0'}
-            categoryTitle={
-              <div className="flex items-center w-full gap-4">
-                <FiInfo size={18} />
-                <p>{Intl.t('password-prompt.sign.description')}</p>
-              </div>
-            }
-          >
-            <AccordionContent customClass={'px-0 pt-4 pb-0'}>
-              <div className="max-w-full overflow-hidden">
-                <p>{Description}</p>
-              </div>
-            </AccordionContent>
-          </AccordionCategory>
-        </div>
+        <>
+          <hr className="h-0.25 bg-neutral opacity-40 w-full" />
+
+          <div className="flex flex-col w-full h-fit">
+            <AccordionCategory
+              isChild={false}
+              iconOpen={<FiChevronDown />}
+              iconClose={<FiChevronUp />}
+              customClass={'!p-0'}
```

Comment:
Do we need to force here!? It should not be used.

Reply:
![Screenshot from 2023-10-10 10-36-51](https://github.com/massalabs/station-massa-wallet/assets/90157528/46ea7820-4669-417b-b643-bfc6de840a2f)
for some odd reason yes

Reply:
![Screenshot from 2023-10-10 10-39-06](https://github.com/massalabs/station-massa-wallet/assets/90157528/f3948758-1f6f-42b1-abb1-51d7846ad1aa)
the custom class is set correctly too

Reply:
but setting it to px-0 and py-0 seems to work correctly

Reply:
Better! 
Force with `!` should be avoided. ;)

Tks for that!